utils/liftover.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+from gnomad_hail.resources import *
+from gnomad_hail.utils.generic import flip_base 
+import logging
+from os.path import dirname, basename
+import sys
+
+
+logging.basicConfig(format="%(asctime)s (%(name)s %(lineno)s): %(message)s", datefmt='%m/%d/%Y %I:%M:%S %p')
+logger = logging.getLogger("liftover")
+logger.setLevel(logging.INFO)
+
+
+def get_checkpoint_path(gnomad: bool, data_type: str, table_path: str) -> str:
+    """
+    Creates a checkpoint path for Table
+    :param bool gnomad: Whether data is gnomAD data
+    :param str data_type: Data type (exomes or genomes for gnomAD; not used otherwise)
+    :param str table_path: Path to input Table (if data is not gnomAD data)
+    :return: Output checkpoint path
+    :rtype: str
+    """
+    
+    if gnomad:
+        return get_gnomad_liftover_data_path(data_type, version=CURRENT_RELEASE)
+    else:
+        out_name = basename(table_path).split('.')[0]
+        return f'{dirname(table_path)}/{out_name}_lift.ht'
+
+
+def lift_ht(ht: hl.Table, gnomad: bool, data_type: str, table_path: str,  rg: hl.genetics.ReferenceGenome) -> hl.Table:
+    """
+    Lifts input Table from one reference build to another
+    :param Table ht: Table
+    :param bool gnomad: Whether data is gnomAD data
+    :param str data_type: Data type (exomes or genomes for gnomAD; not used otherwise)
+    :param str table_path: Path to input Table (if data is not gnomAD data)
+    :param ReferenceGenome rg: Reference genome
+    :return: Table with liftover annotations
+    :rtype: Table
+    """
+
+    # annotate release file with liftover coordinates
+    ht = ht.annotate(new_locus=hl.liftover(ht.locus, rg, include_strand=True),
+                     old_locus=ht.locus
+                    )
```

Comment:
Do you not want to filter out loci that didn't liftover? `hl.is_defined(ht.new_locus)`

Reply:
Good point - I think I'll count the number that didn't liftover and then remove